frameworks/hdfs/docs/install.md

```diff
@@ -68,199 +68,28 @@ The following describes the most commonly used features of DC/OS Apache HDFS and
+The service configuration object contains some properties that MUST be specified during installation and CANNOT be modified after installation is in progress.
 
-```json
-{
-  "service": {
-    "name": "{{ data.serviceName }}",
-    "service_account": "{{ data.serviceName }}-principal",
-  }
-}
-```
-
-<table class="table">
-  <tr>
-    <th>Property</th>
-    <th>Type</th>
-    <th>Description</th>
-  </tr>
-
-  <tr>
-    <td>name</td>
-    <td>string</td>
-    <td>The name of the HDFS service installation. This must be unique for each DC/OS service instance deployed on a DC/OS cluster. It will determine the ID of the HDFS service. See [Alternate install configurations](#alternate-install-configurations) for more information on configuring the service name.</td>
-  </tr>
-
-  <tr>
-    <td>service_account</td>
-    <td>string</td>
-    <td>The DC/OS service account for the HDFS cluster.</td>
-  </tr>
-
-</table>
+For more information on service configuration see [the documentation on configuring a service from the catalog](https://docs.mesosphere.com/latest/deploying-services/config-universe-service/).
 
 ### Node Configuration
 
 The node configuration objects correspond to the configuration for nodes in the HDFS cluster. Node configuration MUST be specified during installation and MAY be modified during configuration updates. All of the properties except `disk` and `disk_type` MAY be modified during the configuration update process.
 
-Example node configuration:
-```json
-  "journal_node": {
-    "cpus": 0.5,
-    "mem": 4096,
-    "disk": 10240,
-    "disk_type": "ROOT",
-    "strategy": "parallel"
-  },
-    "name_node": {
-    "cpus": 0.5,
-    "mem": 4096,
-    "disk": 10240,
-    "disk_type": "ROOT"
-  },
-  "zkfc_node": {
-    "cpus": 0.5,
-    "mem": 4096
-  },
-  "data_node": {
-    "count": 3,
-    "cpus": 0.5,
-    "mem": 4096,
-    "disk": 10240,
-    "disk_type": "ROOT",
-    "strategy": "parallel"
-  }
-```
-
-<table class="table">
-
-  <tr>
-    <th>Property</th>
-    <th>Type</th>
-    <th>Description</th>
-  </tr>
-
-   <tr>
-    <td>cpus</td>
-    <td>number</td>
-    <td>The number of cpu shares allocated to the node's process.</td>
-  </tr>
+#### A Note on Memory Configuration
 
-  <tr>
-    <td>mem</td>
-    <td>integer</td>
-    <td>The amount of memory, in MB, allocated to the node. This value MUST be larger than the specified max heap size. Make sure to allocate enough space for additional memory used by the JVM and other overhead. A good rule of thumb is allocate twice the heap size in MB for memory.</td>
-  </tr>
+As part of the service (or node) configuration, the amount of memory in MB allocated to the node can be specified. This value *must& be larger than the specified maximum heap size. Make sure to allocate enough space for additional memory used by the JVM and other overhead. A good rule of thumb is allocate twice as much memory as the size of the heap.
```

Comment:
I would mention the heap size setting explicitly here to clarify, is it `hdfs.hadoop_heapsize`? I think it would also aid understanding to mention that this constraint holds for every node type. Also, typo with `*must&`.

Reply:
It's a little more complex than that since there are multiple ways to specify the heap size. It can either be done using `hdfs.hadoop_heapsize` for *all* node types, or by the individual `hadoop_<nodetype>_opts`. I have updated the text to make this a bit clearer. Please take a look again @mrbrowning.